### ISSUE TITLE
[MU4 Issue] Keyboard focus at menu is persistent after clicks

### ISSUE THREAD

**Issue by j824h** [musescore/MuseScore#12899]
**Describe the bug**
Keyboard focus at menu created by 'Alt' is not cancelled even after clicks.

**To Reproduce**
(At the main window, after clicking on any area other than the menu)
1. Press 'Alt'
2. Click on anything on the window, any times

**Expected behavior**
Keyboard focus created at 1. is cancelled at 2.

**Screenshots**
![image](https://user-images.githubusercontent.com/10954732/185036648-a58fab3d-4579-4171-abd0-8427d31d78b9.png)

**Platform information**
 - OS: Linux

**Additional context**
This behavior frequently tricks me after I pressed Alt (accidentally / changing mind to combo a shortcut).
I can notice the focus stayed there only after getting surprised by pressing Space to play/pause and ending up opening a new window.

Similarly, pressing (Alt +) Tab after 1. creates another keyboard focus visualized simultaneously, not actually effective at the moment.
Pressing (Alt +) Tab once more cancels the keyboard focus at the menu, and then tab navigation works as expected.

Again similarly, pressing a key that doesn't correspond to any menu shortcut (e.g. 'c' at the top level menu), works as a general keyboard shortcut and doesn't cancel the keyboard focus.

Generally, the keyboard focus at menu is now too persistive.
I feel the keyboard navigation for the menu should stay more like in MU3.

**Comment by MarcSabatella** [comment]
Yes, I've noticed this as well, but couldn't quite put into words what was happening!

BTW, the workaround I normally use is to press Esc.  This leaves the menu bar and returns focus to the score, if that's where it last was.  If you've already pressed a letter and opened a menu, you'll need to press Esc twice to close the menu and leave the menu bar.  Just don't press three times or you lose your selection.  But even then, Shift+L can partially restore it.

**Comment by j824h** [comment]
https://github.com/Eism/MuseScore/commit/5bb5da51a025e38451f64de5347d6164558779fb
This makes clicking cancel.

However, pressing a (Shift + ) Tab still doesn't cancel at once.
Also, pressing a key that doesn't correspond to any menu shortcut still doesn't cancel.

Should I open separate issues for those related issues?

**Comment by DmitryArefiev** [comment]
Fixed in #13009

**Comment by DmitryArefiev** [comment]
> Much thanks to [Eism@5bb5da5](https://github.com/Eism/MuseScore/commit/5bb5da51a025e38451f64de5347d6164558779fb) ! This makes clicking cancel.
> 
> However, pressing a (Shift + ) Tab still doesn't cancel at once. Also, pressing a key that doesn't correspond to any menu shortcut still doesn't cancel.
> 
> Should I open separate issues for those related issues?

@j824h Yes, please open a separate issue for and mention it here.

Also, I checked briefly and looks like those issues are left on Linux only.

Thanks!
